Formula/openexr.rb

```diff
@@ -2,8 +2,8 @@ class Openexr < Formula
   desc "High dynamic-range image file format"
   homepage "https://www.openexr.com/"
   # NOTE: Please keep these values in sync with imath.rb when updating.
```

Comment:
do we still need this?

Reply:
I don't think so. The latest version of `imath` is 3.1.9.

Reply:
we can followup removing the comment.